gapis/shadertools/cc/spv_manager.cpp

```diff
@@ -112,6 +112,18 @@ void SpvManager::renameViewIndex() {
   }
 }
 
+// Remove all layout(location = ...) qualifiers.
+// For most use cases we should be binding/remapping locations regardless whether they
+// are assigned explicit location or compiler generated one.
+// TODO: Handle separate shader objects - we just hope the game uses consistent block names for now.
+bool SpvManager::removeLayoutLocations() {
```

Comment:
I started that, but actually finding if the variable is a block or not is non-trivial, and we do not want to rename non-blocks (for now, I wouldn't entirety rule it out)

The question, would such renaming scheme work 100% of the time?